### ISSUE TITLE
How crop image not in center. 

### ISSUE THREAD

**Issue by Lucuz** [cubicdaiya/ngx_small_light#28]
Hi, I have a question. 
I want crop a image but the focal point is not in center of image, is possible to crop not in center? 
This is a sample image
![husky-15072-2560x1600](https://cloud.githubusercontent.com/assets/10074650/8511034/6c6849b4-2305-11e5-9259-8c2fd5afd3f7.jpg)
And this Is the result I want 
![husky-15072-2560x1600-1](https://cloud.githubusercontent.com/assets/10074650/8511039/9f1628d6-2305-11e5-9747-5d0780090fc9.jpg)

**Comment by cubicdaiya** [comment]
No. Now `ngx_small_light` does not support the flexible cropping very well. But patches welcome.
